File: models/domain/Account.py
from flask_restful import fields


class Account():
    resource_fields = {
        'id': fields.Integer,
        'name': fields.String,
        'description': fields.String,
        'color': fields.String,
    }

    def __init__(self, name=None, description=None, color=None):
        self.name = name
        self.description = description
        self.color = color

File: frontend/src/app/components/accounts/account.ts
export class Account {

  public id: number;
  public name: string;
  public description: string;
  public total: number;
  public color: string;
  public notify: boolean;
  public lastUpdate: Date;

  constructor(id: number, name: string, description: string, total: number, color: string, notify: boolean) {
    this.id = id;
    this.name = name;
    this.description = description;
    this.total = total;
    this.color = color;
    this.notify = notify;
  }
}
